calico/felix/felix.py

```diff
@@ -273,10 +276,10 @@ def handle_endpointcreated(self, message):
         """
         log.debug("Received endpoint create: %s", message.fields)
 
+        # TODO: Ought to firewall missing mandatory fields here.
         endpoint_id = message.fields['endpoint_id']
-        resync_id   = message.fields['resync_id']
-        issued      = message.fields['issued']
         mac         = message.fields['mac']
+        resync_id   = message.fields.get('resync_id')
```

Comment:
I was going with being tolerant of missing fields, at least for now. There's a TODO above to tidy this stuff up, and put in some better handling of missing fields.